src/raftstore/store/worker/metrics.rs

```diff
@@ -74,4 +74,15 @@ lazy_static! {
             "tikv_raftstore_hash_duration_seconds",
             "Bucketed histogram of raftstore hash compution duration"
         ).unwrap();
+
+    pub static ref SNAPSHOT_KV_COUNET_GAUGE: Gauge =
```

Comment:
Should use Histogram instead. Ref: https://prometheus.io/docs/concepts/metric_types/